Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -108,13 +114,16 @@ void HiresTexture::Update()
 
     if (FileName.substr(0, code.length()) == code)
     {
-      s_textureMap[FileName] = rFilename;
+      s_textureMap[FileName] = {rFilename, false};
       s_check_native_format = true;
     }
 
     if (FileName.substr(0, s_format_prefix.length()) == s_format_prefix)
     {
-      s_textureMap[FileName] = rFilename;
+      const size_t arb_index = FileName.rfind("_arb");
+      if (arb_index != std::string::npos)
+        FileName.erase(arb_index, 4);
+      s_textureMap[FileName] = {rFilename, arb_index != std::string::npos};
```

Comment:
Really minor nitpick, but what about `const bool has_arbitrary_mipmaps = arb_index != std::string::npos;` and using that in both places?
(tbf what bothers me more when looking at this is the mix of `filenames`, prefixed `rFilename` and first-upper-cased `FileName` :sweat_smile:)